src/main/java/org/zalando/nakadi/domain/SchemaChange.java

```diff
@@ -2,26 +2,42 @@
 
 public class SchemaChange {
     public enum Type {
-        ID_CHANGED,
-        DESCRIPTION_CHANGED,
-        TITLE_CHANGED,
-        PROPERTIES_ADDED,
-        SCHEMA_REMOVED,
-        TYPE_CHANGED,
-        NUMBER_OF_ITEMS_CHANGED,
-        PROPERTY_REMOVED,
-        DEPENDENCY_ARRAY_CHANGED,
-        DEPENDENCY_SCHEMA_CHANGED,
-        COMPOSITION_METHOD_CHANGED,
-        ATTRIBUTE_VALUE_CHANGED,
-        ENUM_ARRAY_CHANGED,
-        SUB_SCHEMA_CHANGED,
-        DEPENDENCY_SCHEMA_REMOVED,
-        REQUIRED_ARRAY_CHANGED,
-        REQUIRED_ARRAY_EXTENDED,
-        ADDITIONAL_PROPERTIES_CHANGED,
-        ADDITIONAL_PROPERTIES_NARROWED,
-        ADDITIONAL_ITEMS_CHANGED
+        ID_CHANGED(Version.Level.MAJOR),
+        DESCRIPTION_CHANGED(Version.Level.PATCH),
+        TITLE_CHANGED(Version.Level.PATCH),
+        PROPERTIES_ADDED(Version.Level.MINOR),
+        SCHEMA_REMOVED(Version.Level.MAJOR),
+        TYPE_CHANGED(Version.Level.MAJOR),
+        NUMBER_OF_ITEMS_CHANGED(Version.Level.MAJOR),
+        PROPERTY_REMOVED(Version.Level.MAJOR),
+        DEPENDENCY_ARRAY_CHANGED(Version.Level.MAJOR),
+        DEPENDENCY_SCHEMA_CHANGED(Version.Level.MAJOR),
+        COMPOSITION_METHOD_CHANGED(Version.Level.MAJOR),
+        ATTRIBUTE_VALUE_CHANGED(Version.Level.MAJOR),
+        ENUM_ARRAY_CHANGED(Version.Level.MAJOR),
+        SUB_SCHEMA_CHANGED(Version.Level.MAJOR),
+        DEPENDENCY_SCHEMA_REMOVED(Version.Level.MAJOR),
+        REQUIRED_ARRAY_CHANGED(Version.Level.MAJOR),
+        REQUIRED_ARRAY_EXTENDED(Version.Level.MINOR, Version.Level.MAJOR),
```

Comment:
`REQUIRED_ARRAY_EXTENDED` used to be `MINOR` for `forward` but `major` for `compatible`.

Reply:
@rcillo Is it changed?

Reply:
I mean it is exactly as it was: MINOR for forward and MAJOR for compatible.

Reply:
😞 my bad, I did not read it through.